sdk/vaa/payloads.go

```diff
@@ -250,6 +257,27 @@ func (r BodyWormchainMigrateContract) Serialize() []byte {
 	return serializeBridgeGovernanceVaa(WasmdModuleStr, ActionMigrateContract, ChainIDWormchain, r.MigrationParamsHash[:])
 }
 
+func (r BodyWormchainAllowlistInstantiateContract) Serialize() []byte {
+	payload := &bytes.Buffer{}
+	payload.Write(r.ContractAddr[:])
+	MustWrite(payload, binary.BigEndian, r.CodeId)
+	return serializeBridgeGovernanceVaa(WasmdModuleStr, ActionAllowlistInstantiateContract, ChainIDWormchain, payload.Bytes())
+}
+
+func (r *BodyWormchainAllowlistInstantiateContract) Deserialize(bz []byte) {
+	if len(bz) != 40 {
+		panic("incorrect payload length")
+	}
+
+	var contractAddr [32]byte
+	copy(contractAddr[:], bz[0:32])
```

Comment:
Should there be some correlation between uint64 and only 8 digits for it to fit into?  Like should the CodeId be uint16 or should there be more than 8 digits to hold a uint64?  Or am I just missing something?  It's possible the encoding/decoding enforces the codeId fits into 8 places and I just missed it.

Reply:
The uint64 is 64 bits = 8 bytes, correct?

Since the payload is a byte array, I'm formatting it as: 1) first 32 bytes are for the contract address, 2) last 8 bytes are for the codeId.